visallo/visallo
# Don't use the Maven Shade Plugin when Packaging GPW jar Files

With the LibDirectoryLoader and the HdfsLibLoader both being available to the GraphPropertyRunner, there is no need to include all dependencies in the GPW jars.


Testing Instructions:
1. Deploy the GPW jars
2. Ensure no errors when starting
3. Ensure that when data is entered into the system the GPW's all perform as expected

CHANGELOG
Changed: GPW jar files no longer contain all dependencies
